bin/proxy.py

```diff
@@ -52,7 +52,7 @@ def load_translations():
     translations = {}
 
     for f_name in PROXY_LANGS:
-        f_name = 'share/translations/%s.txt' % f_name
+        f_name = '/root/app/share/translations/%s.txt' % f_name
```

Comment:
this will only work in the docker image b/c you're using absolute paths

Reply:
Ok absolute paths removed in place of computed paths :)